strr-web/components/bcros/form-section/BusinessDetails.vue

```diff
@@ -1,26 +1,28 @@
 <template>
-  <div data-test-id="property-manager-business-details" class="with-divider">
-    <BcrosFormSection :title="t('createAccount.propertyManagerForm.businessDetails')">
-      <div class="flex flex-row justify-between w-full mb-[40px] mobile:flex-col mobile:mb-[16px]">
+  <div data-test-id="form-section-business-details">
+    <BcrosFormSection :title="t('common.formLabels.businessDetails')">
+      <div class="flex flex-row justify-between w-full mb-10 m:flex-col m:mb-4">
         <UFormGroup name="businessName" class="flex-grow" :error="errors.businessLegalName">
           <UInput
             v-model="businessName"
             type="text"
-            :placeholder="t('createAccount.propertyManagerForm.businessName')"
-            data-test-id="property-manager-business-name-input"
+            :placeholder="isBusinessNameRequired
+              ? t('common.formLabels.businessLegalName')
```

Comment:
@dimak1 Re validation - are you enforcing when it is a required field.

Reply:
When the `contactType` is Business, this field is required. So I created this prop to force the input.

Reply:
All good, can we resolve it?